service/matching/db.go

```diff
@@ -165,13 +186,20 @@ func (db *taskQueueDB) renewTaskQueueLocked(
 	return nil
 }
 
-// UpdateState updates the queue state with the given value
+// UpdateState updates the queue state with the given values
```

Comment:
There is only one parameter to this function, so shouldn't it still be "value"?
```suggestion
// UpdateState updates the queue state with the given value
```